mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTCommonConfiguration.java

```diff
@@ -168,6 +168,20 @@
     )
     private int mqttProxyNumIOThreads = Runtime.getRuntime().availableProcessors();
 
+    @FieldContext(
+            category = CATEGORY_MQTT_PROXY,
+            doc = "Maximum number of lookup requests allowed on "
+                    + "each broker connection to prevent overloading a broker."
+    )
+    private int mqttProxyClientMaxLookupRequest = 50000;
+
+    @FieldContext(
+            category = CATEGORY_MQTT_PROXY,
+            doc = "The number of concurrent lookup requests that can be sent on each broker connection. "
+                    + "Setting a maximum prevents overloading a broker."
+    )
+    private int mqttProxyClientConcurrentLookupRequest = 5000;
```

Comment:
It's better to move these two configs to `MQTTProxyConfiguration`.
And also rename to `maxLookupRequest`, `concurrentLookupRequest`

Reply:
fine! i have updated